internal/workflow.go

```diff
@@ -59,6 +59,23 @@ const (
 	HandlerUnfinishedPolicyAbandon
 )
 
+// VersioningBehavior specifies when existing workflows could change their Build ID.
+// NOTE: Experimental
+type VersioningBehavior int
+
+const (
+	// Workflow versioning policy unknown. A default VersioningBehaviorUnspecified policy forces
+	// every workflow to explicitly set a VersioningBehavior different from VersioningBehaviorUnspecified.
+	VersioningBehaviorUnspecified VersioningBehavior = iota
```

Comment:
Hrmm, I wonder if we want to expose the raw proto enum like we do with `StartWorkflowOptions.WorkflowIDConflictPolicy` or not like I guess we chose not with `WorkflowUpdateStage`?

Reply:
We could, what do we want to do moving forward @Quinn-With-Two-Ns , wrap enums or raw protos?

Reply:
Yeah there is pros and cons to both and the SDK has historically not been consistent. I would bias towards wrapping the enums.

Reply:
Works for me